### PR TITLE
Cypress test for issue 1870

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add Cypress test for issue [1870](https://github.com/opencv/cvat/issues/1870)

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            cy.createShape(309, 431, 616, 671)
+        })
+        it('Check what cursor doesn\'t automatically jumps at the end of the word', () => {
+            cy.get('.ant-row > .ant-collapse > .ant-collapse-item > .ant-collapse-header > span')
+            .should('contain', 'Details')
+            .click()
+            cy.get('.ant-input')
+            .type('{leftarrow}{leftarrow}ee')
+            cy.get('.ant-input')
+            .should('have.value', newLabelAttrValue)
```
Comment (bsekachev) [inline tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js:54]: Can we unite it to the same chain without extra ``cy.get()``?

File: tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        })
+        it('Open a job', () => {
+            cy.contains('a', 'Job #').click()
+            cy.url().should('include', '/jobs')
+        })
+        it('Create a shape', () => {
+            cy.createShape(309, 431, 616, 671)
+        })
+        it('Check what cursor doesn\'t automatically jumps at the end of the word', () => {
+            cy.get('.ant-row > .ant-collapse > .ant-collapse-item > .ant-collapse-header > span')
```
Comment (bsekachev) [inline tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js:48]: Not really reliable selector. It is better to use an identificator before. Here can be a lot of collapses in the future.

File: tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js]: Yes, we can. I've reworked this part of the code a bit.

File: tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/issue_1870_cursor_not_jump_to_end.js]: I tried to shorten selector.

**Pull request merged**